packages/lib/slugify.ts

```diff
@@ -1,6 +1,9 @@
-export const slugify = (str: string) => {
+export const slugify = (str: string, toLowerCase?: boolean) => {
+  if (toLowerCase !== false) {
+    str = str.toLocaleLowerCase(); // Convert to lowercase
```

Comment:
why are you using toLocaleLowerCase() here instead of toLoweCase() ?

Reply:
ahh! was fiddling with another project requiring localizations that got mixed up here, have rectified it please review.